utils/e2e-tests/bash/fixtures/help.stdout.txt

```diff
@@ -191,44 +160,10 @@ Options:
           - log: Output the tracing records using the log
 
       --state-pruning <PRUNING_MODE>
-          Specify the state pruning mode.
-          
-          This mode specifies when the block's state (ie, storage) should be pruned (ie, removed) from the database.
-          
-          This setting can only be set on the first creation of the database. Every subsequent run will load the pruning mode from the database and will error if the stored mode doesn't match this CLI value. It is fine to drop this CLI flag for subsequent runs.
-          
-          Possible values:
-          
-          - archive:
-          
-          Keep the state of all blocks.
-          
-          - 'archive-canonical'
-          
-          Keep only the state of finalized blocks.
-          
-          - number
-          
-          Keep the state of the last number of finalized blocks.
-          
-          [default: 256]
+          Specify the state pruning mode. This mode specifies when the block's state (ie, storage) should be pruned (ie, removed) from the database. This setting can only be set on the first creation of the database. Every subsequent run will load the pruning mode from the database and will error if the stored mode doesn't match this CLI value. It is fine to drop this CLI flag for subsequent runs. Possible values: - archive: Keep the state of all blocks. - 'archive-canonical' Keep only the state of finalized blocks. - number Keep the state of the last number of finalized blocks. [default: 256]
```

Comment:
This looks broken, we should see if there's a fix available

Reply:
Just checked in `substrate: polkadot-v0.9.42`, the same help stdout.

In our `humanode` codebase we use the latest clap versions `4.5.4`.